### DISCUSSION TITLE
ZEALOT_DOMAIN变化后应该怎么生效

### DISCUSSION THREAD

**Discussion by Cabriter** [tryzealot/zealot#724] [Q&A] [answered]
服务器地址变了,IP变了,应该如何让ZEALOT_DOMAIN变更为新的服务器ip地址?

**Comment by Cabriter** [comment]
@icyleaf

**Comment by icyleaf** [comment]
ZEALOT_DOMAIN 不绑定任何 IP 的呀，他只是负责服务绑定链接的外部显示

**Reply by Cabriter** [reply]
我目前遇到一个问题,服务器ip变了,导致下载apk的时候url是以前的ip拼起来的url,然后下载不到apk.我认为应该是ZEALOT_DOMAIN与实际服务器ip不一致导致的.

**Reply by Cabriter** [reply]
![image](https://user-images.githubusercontent.com/12536807/157160392-ee103dc3-fc51-4252-99e5-911472564e21.png)

**Comment by icyleaf** [comment, marked as answer]
那说明你当时设置的 ZEALOT_DOMAIN 是 IP 地址，建议在环境变量文件修改后重启，临时解决可在系统设置修改站点网站域名(Site Domain)，这个会写入到数据库但不会覆盖环境变量
